package.json

```diff
@@ -82,7 +84,7 @@
   },
   "homepage": "https://github.com/BrewInteractive/nestjs-hasura#readme-module",
   "dependencies": {
-    "graphql-request": "^7.0.1"
+    "graphql-request": "^6.1.0"
```

Comment:
Why older version?

Reply:
Downgraded due to a bug in the new version.